lm_eval/models/octoai_llms.py

```diff
@@ -3,9 +3,9 @@
 import time
 
 import requests
```

Comment:
Do we need requests package now?

Reply:
yes, because in single batch mode requests package is used